ccd-definition/CaseEventToComplexTypes/noticeOfProceedings-JudgeAndLegalAdvsor-SDO_READY_TO_SEND.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
That file contains typo, but not massively important.

Reply:
Ups, copy-paste of other file of same name. Will fix it before merge. Good spot.